src/ui/main_window.ui

```diff
@@ -359,6 +360,14 @@
     <string>Alt+V</string>
    </property>
   </action>
+  <action name="dactionInputLine">
+   <property name="text">
+    <string>Compact input line</string>
+   </property>
+   <property name="shortcut">
+    <string>Alt+J</string>
```

Comment:
Why `J` by the way? It is a bit difficult to include it with a menu item as an accelerator even in the native en_US..! `L` strikes me as a bit easier to handle (so that the second above line would become: `    <string>Compact input &line</string>` if I have got that right) any thoughts...? 😉

Of course the more accelerators/short-cuts we hard code into the application the more the likely-hood that they will clash with an OS one or prevent the user assigning them to a _Key-binding_ - 🙁

Reply:
Sounds OK to me.

Reply:
Changed to Alt+L. Don't want to add the underline though, it already says Alt+L on the side - gets the point across.